datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -356,13 +356,6 @@ fn rountrip_aggregate() -> Result<()> {
             Vec::new(),
             Vec::new(),
         ))],
-        // STRING_AGG
-        vec![Arc::new(StringAgg::new(
```

Comment:
I think we should create expression with `udaf::create_aggregate_expr`

Reply:
Added.